ast.ts

```diff
@@ -14,7 +14,7 @@ export type Parameter<A> = { name: string, type: Type }
 
 export type Program<A> = { a?: A, funs: Array<FunDef<A>>, inits: Array<VarInit<A>>, classes: Array<Class<A>>, stmts: Array<Stmt<A>> }
 
-export type Class<A> = { a?: A, name: string, fields: Array<VarInit<A>>, methods: Array<FunDef<A>>}
+export type Class<A> = { a?: A, name: string, supers?: Array<string>, fields: Array<VarInit<A>>, methods: Array<FunDef<A>>}
```

Comment:
We made it optional because we were afraid of breaking other people's parsers before.
Now that we have a better understanding of how we will try to merge everyone's changes in, it doesn't have to be optional anymore.